activitysim/abm/models/cdap.py

```diff
@@ -36,12 +36,16 @@ class CdapSettings(PydanticReadable, extra="forbid"):
     JOINT_TOUR_COEFFICIENTS: str = "cdap_joint_tour_coefficients.csv"
     JOINT_TOUR_USEFUL_COLUMNS: list[str] | None = None
     """Columns to include from the persons table that will be need to calculate household joint tour utility."""
-    annotate_persons: PreprocessorSettings | None = None
-    annotate_households: PreprocessorSettings | None = None
     COEFFICIENTS: Path
     CONSTANTS: dict[str, Any] = {}
     compute_settings: ComputeSettings | None = None
 
+    preprocessor: PreprocessorSettings | None = None
+    """Preprocess choosers tables before running the model."""
+    annotate_persons: PreprocessorSettings | None = None
```

Comment:
Automatic documentation tools will work better if `annotate_persons` has its own docstring, even though it's clear to a human reader the one docstring below applies to both.

Reply:
done